client/src/app/login/login.component.scss

```diff
@@ -20,4 +20,8 @@ input[type=submit] {
 .create-an-account, .forgot-password-button {
   color: #000;
   cursor: pointer;
+  
+  &:hover {
+    text-decoration: none;
```

Comment:
I think it's important to keep it (and add it to the forgot password button) for accessibility

Reply:
Okay, either way it was just for consistency.